concepts/problem-solving/README.md

```diff
@@ -1,96 +1,149 @@
-How long did it take to isolate the bug?  
-Did I stay calm?  
-Did my mindset help/hinder me?  
+If you've been problem solving for over 15 mins and you feel like you've hit a dead end, post a message in your cohort Slack channel. It's likely someone else has had and resolved a similar issue, or has the perspective, understanding or experience to help you resolve your issue.
 
+Students are sometimes reluctant to do this, for a variety of reasons:
+ * "I don't want to take up their time"
+ * "I don't them to think I'm not as smart as they are"
+ * "I'm ashamed of how long this is taking me"
+ * "I've already asked for help three times this week!"
 
-## Prerequisites
+It's human nature to want to be independent, but this attitude can sometimes slow you down, and in an industry context it can slow down an entire team. Consider, what if by remaining quiet you're denying another team member the opportunity to become a better teacher? What if the answer is something you couldn't possibly have known, but another developer has learned only through long and painful experience? What if someone else has the exact same problem and could benefit from your speaking up?
 
-None
+What if the team becomes stronger when we admit we don't know something?
 
-## Capabilities
-You are comfortable:
+We like to emphasise the team as a powerful tool for problem solving because we want teams in the industry to be like this, and we believe that the first step is to train each other to reach out, to depend on each other, to be greater than the sum of their parts.
 
- - perservering with a tough problem  
- - using strategies to solve a tough problem  
- - recognising when it's time to ask for help  
+### 10. Still blocked? Tag a coach
+We recognise that there are some problems that can't be solved within the context of the team. If your cohort can't help you, or you feel a concept needs more clarification, tag a teacher in Slack. We'll try to help out as soon as possible. To be of most help, there are a few things you can do to reduce the time it takes us to understand your problem:
+
+1. _Format your code_. Did we mention that already? Maybe once or twice.
+2. Don't post screenshots of code. Screenshots are pretty but can't be copied, pasted, or scrolled. Post snippets, excerpts surrounded by triple backtick, [gists](https://gist.github.com), or just a link to the file in your repository on GitHub.
+3. Explain simply and clearly:
+  * What you _expect_ the code to do
+  * What it _actually_ does
+  * Any error messages you receive
+
+### 11. Reflect  
+
+Sometimes it's tempting to charge ahead, steamroller-like, once you've finally managed to kick that pesky problem to the curb. We suggest, first and foremost, you're probably due a break! Stand up, relax, stretch, breathe. And consider:
+ * Which approaches worked well?    
+ * What did I get stuck on?   
+ * Why did it take so long to resolve a silly bug? A common question!   
```

Comment:
Could add: What were the troubleshooting strategies/patterns that helped (or could have helped) that I should consider for the future?